### PR TITLE
Brand changed

### PR DESCRIPTION
**Pull request by ferranpons**
We have adapted the logo, styles and sample to fit Schibsted branding style.

### CODE REVIEW

File: app/src/main/res/drawable/leku_app_button_border.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+       android:shape="rectangle">
+  <corners android:radius="5dp" />
+  <stroke android:width="3dp" android:color="@color/leku_app_blue" />
+</shape>
```
Comment (AlbertVilaCalvo) [inline app/src/main/res/drawable/leku_app_button_border.xml]: new line

File: app/src/main/res/values/colors.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <color name="leku_app_blue">#3DA8E1</color>
+  <color name="leku_app_strong_blue">#053B58</color>
+</resources>
```
Comment (AlbertVilaCalvo) [inline app/src/main/res/values/colors.xml]: new line

File: app/src/main/res/drawable/leku_app_button_border.xml (hunk 1 above)
Comment (ferranpons) [reply app/src/main/res/drawable/leku_app_button_border.xml]: done!

File: app/src/main/res/values/colors.xml (hunk 2 above)
Comment (ferranpons) [reply app/src/main/res/values/colors.xml]: done!

**Pull request merged**